src/uwtools/resources/jsonschema/files-to-stage.jsonschema

```diff
@@ -0,0 +1,7 @@
+{
+  "additionalProperties": {
+    "type": "string"
+  },
+  "minProperties": 1,
+  "type": "object"
+}
```

Comment:
Currently only used by `fv3` because `sfc_climo_gen` can refer to all its input files via absolute paths in its `config` namelist, but certain to be used by other drivers.